### PR TITLE
fix to correct archive filesize

### PR DESCRIPTION
**Pull request by blankdots**

### CODE REVIEW

File: cmd/ingest/ingest.go [hunk 1]
```diff
@@ -201,7 +201,7 @@ func main() {
 			log.Debugln("Mark as archived")
 			fileInfo := database.FileInfo{}
 			fileInfo.Path = archivedFile
-			fileInfo.Size = fileSize
+			fileInfo.Size, err := archive.GetFileSize(archivedFile)
```
Comment (jbygdell) [inline cmd/ingest/ingest.go:204]:
```suggestion
			fileInfo.Size, err := archive.GetFileSize(archivedFile)
			if err != nil {
				log.Errorf("Failed to get file size of: %s, reason: %v", archivedFile, err)
				continue
			}
```

**Review by jbygdell — CHANGES_REQUESTED** [review]
Need to catch the error

File: cmd/ingest/ingest.go (hunk 1 above)
Comment (blankdots) [reply cmd/ingest/ingest.go]: my editor complains about `:=` so `=` seems to work

**Comment by blankdots** [comment]
I would approve but I cannot as i started this, so I think the changes are ok and we can merge.

**Comment by pontus** [comment]
I think only admin can approve as long as anyone (e.g. @jbygdell) has requested changes.

**Review by jbygdell — APPROVED** [review]

**Pull request merged**